src/node.cc

```diff
@@ -3751,7 +3752,9 @@ static void ParseArgs(int* argc,
       default_cipher_list = arg + 18;
     } else if (strncmp(arg, "--use-openssl-ca", 16) == 0) {
       ssl_openssl_cert_store = true;
+      use_ca_flag |= 1;
```

Comment:
Unless there's a big speedup with this, I'd rather just have the flags, seems clearer